src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+    removeBook: (state, action) => ({
+      ...state,
+      books: state.books.filter(
+        (book) => book.id !== action.payload,
+      ),
+    }),
+  },
+});
+
+export default booksSlice.reducer;
```

Comment:
Well done @preciousbetine 

You did everything well except that I do not see how you exported the actions using the ES6 object destructuring from `booksSlice.actions`. 

```javascript
export const { addBook, removeBook } = booksSlice.actions
```

Kindly add it